yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -163,6 +163,24 @@ func (d *TableIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 						))
 				}
 			}
+
+			if c.ConstraintType == queryparser.PRIMARY_CONSTR_TYPE {
+				//Report PRIMARY KEY (createdat timestamp) as hotspot issue
+				for idx, col := range c.Columns {
```

Comment:
Do we really need a for-loop here? We just want to check the first column, right?

Reply:
yeah, for loop is unneccessary, we just need to check first col. fixing it!